ethcore/src/client/bad_blocks.rs

```diff
@@ -57,8 +58,10 @@ impl BadBlocks {
 						.enumerate()
 						.map(|(index, tx)| format!("[Tx {}] {:?}", index, tx))
 						.join("\n"),
-				);
+				);*/
 				self.last_blocks.write().insert(unverified.header.hash(), (unverified, message));
+				let num: u32 = 3;
+				BlockChainReset::reset(num);
```

Comment:
you marked my previous comment as resolved, without actually answering:

Does this actually compile on your side?

```rust
64  |                 BlockChainReset::reset(num);
    |                 ^^^^^^^^^^^^^^^^^^^^^^^^^^^ expected 2 parameters
```

Reply:
Thanks and no. Do you know which param is missing?

Reply:
This is not about the param. You should simply not send a PR in and document it in a way that it looks verified. This can be very very misleading.

Reply:
I mean:

> Is there something left for follow-up PRs?
> I don't think so.
> Fixes/Related #10085

this gives the impression "all done", but the thing is that the code does not even compile.